test/test_type_attr.erl

```diff
@@ -7,3 +7,6 @@
 
 %% test case for opaque decl
 -opaque int() :: integer().
+
+-record(state, {name :: string(), param :: term()}).
+-type record_as_type(A) :: #state{param :: A}.
```

Comment:
Could you add the purpose of this test-case as a comment? e.g., `%% test case for record type and record field type`

Reply:
I added.